docs/content/docs/install/manual_demo/_index.md

```diff
@@ -10,15 +10,13 @@ weight: 2
 
 The OSM Manual Install Demo Guide is a step by step set of instructions to quickly demo OSM's key features.
 
-## Configure Prerequisites
-
-- Kubernetes cluster running Kubernetes v1.15.0 or greater
-- Have `kubectl` CLI installed - [Install and Set Up Kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
-- kubectl current context is configured for the target cluster install
-  - `kubectl config current-context`
-- Have a local clone of the OSM GitHub Repo
-  - `git clone https://github.com/openservicemesh/osm.git`
-  - `cd osm`
+
+## Prerequisites
+This demo of OSM v0.8.2 requires:
+  - a cluster running Kubernetes v1.15.0 or greater
```

Comment:
```suggestion
  - a cluster running Kubernetes v1.18.0 or greater
```

This will help with consistency across all platforms

Reply:
Upstream support >= 1.17.